### ISSUE TITLE
JS typewriter doesn't display on e.g. Helen's phone (Google 5x)

### ISSUE THREAD

**Issue by DeckOfPandas** [DeckOfPandas/peanut-raspberry#54]

**Comment by DeckOfPandas** [comment]
Typewriter on devices would be ideal because it’s cool, but static text would be fine if impossible/super awkward

Also the counter and date don’t display either

**Comment by DeckOfPandas** [comment]
![screenshot_20171019-112017](https://user-images.githubusercontent.com/2399432/31766499-a732ebb4-b4bf-11e7-8df2-fc7344861181.png)

![screenshot_20171019-112025](https://user-images.githubusercontent.com/2399432/31766502-a8f4d5d4-b4bf-11e7-9364-a501f0eb068b.png)

**Comment by DeckOfPandas** [comment]
Fred said:
```
shrink the logo in the top left via responsive (ie the @include(mobile) scss) in smaller versions and that will fix 54

(or at least let you see the typewriter code
```

**Comment by DeckOfPandas** [comment]
@blu3id accidentally volunteered to have this added to his procrastination list

**Comment by DeckOfPandas** [comment]
Setting max-width doesn't help (logo smaller, but still forces a double line for the navbar). What am I missing?

**Comment by DeckOfPandas** [comment]
Actually I don't think this issue is due to the title being judgment behind the navbar:

![screenshot_20171023-173402](https://user-images.githubusercontent.com/2399432/31901285-e66d08a8-b818-11e7-8244-2ecf8d24c53d.png)

**Comment by DeckOfPandas** [comment]
Yep, more evidence:
![screen shot 2017-10-23 at 18 14 39](https://user-images.githubusercontent.com/2399432/31902828-1d0fa104-b81e-11e7-8454-63ac43cc32b7.png)

**Comment by DeckOfPandas** [comment]
@blu3id says: #cantreplicate #wontfix

**Comment by DeckOfPandas** [comment]
Closed by mistake

**Comment by DeckOfPandas** [comment]
Fixed in a hacky way in #77
